drivers/overlay/check_116.go

```diff
@@ -0,0 +1,39 @@
+// +build go1.16
+
+package overlay
+
+import (
+	"io/fs"
+	"path/filepath"
+	"strings"
+
+	"github.com/containers/storage/pkg/archive"
+	"github.com/containers/storage/pkg/system"
+)
+
+func scanForMountProgramIndicators(home string) (detected bool, err error) {
+	err = filepath.WalkDir(home, func(path string, d fs.DirEntry, err error) error {
```

Comment:
should we handle the `err != nil` case?

Reply:
Evidently, we need to.  Pushing a tweak for that.